app/models/person.rb

```diff
@@ -22,6 +22,20 @@ class Person < ApplicationRecord
 
   validate :preferred_contact_method_present!
 
+  def self.create_from_peer_to_peer_params!(current_user, peer_to_peer_params)
```

Comment:
Minor preference for expanding `peer_to_peer_params` into keyword arguments.

Also, please add the same fixme comment as above.